File: app/src/main/java/com/tealium/example/MainActivity.java
package com.tealium.example;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.AppCompatButton;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    AppCompatButton mEditUserDetailsButton;
    AppCompatButton mEventsAndAttributesButton;
    AppCompatButton mEditUsetEngagementsButton;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        mEditUserDetailsButton = findViewById(R.id.btn_edit_user_details);
        mEditUserDetailsButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this, UserActivity.class));
            }
        });

        mEventsAndAttributesButton = findViewById(R.id.btn_send_events);
        mEventsAndAttributesButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this, EventsActivity.class));
            }
        });

        mEditUsetEngagementsButton = findViewById(R.id.btn_edit_user_engagement);
        mEditUsetEngagementsButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this, EngagementActivity.class));
            }
        });
    }
}

File: app/src/main/java/com/tealium/example/MainActivity.kt
package com.tealium.example

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.AppCompatButton

class MainActivity : AppCompatActivity() {

    lateinit var editUserDetailsButton: AppCompatButton
    lateinit var eventsAndAttributesButton: AppCompatButton
    lateinit var editUsetEngagementsButton: AppCompatButton
    lateinit var locationButton: AppCompatButton

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        editUserDetailsButton = findViewById(R.id.btn_edit_user_details)
        editUserDetailsButton.setOnClickListener { startActivity(Intent(this@MainActivity, UserActivity::class.java)) }
        eventsAndAttributesButton = findViewById(R.id.btn_send_events)
        eventsAndAttributesButton.setOnClickListener { startActivity(Intent(this@MainActivity, EventsActivity::class.java)) }
        editUsetEngagementsButton = findViewById(R.id.btn_edit_user_engagement)
        editUsetEngagementsButton.setOnClickListener { startActivity(Intent(this@MainActivity, EngagementActivity::class.java)) }
        locationButton = findViewById(R.id.btn_location)
        locationButton.setOnClickListener { startActivity(Intent(this@MainActivity, LocationActivity::class.java)) }
    }
}